quickstart/app/Models/Category.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use App\Models\Category;
+use App\Models\Product;
+
+class Category extends Model
+{
+    protected $table="category";
+    protected $fillable = [
+        'name', 
+        'parent',
+    ];
+
+    public function Product()
```

Comment:
```suggestion
    public function products()
```
chú ý tên function và số nhiều